### PR TITLE
Fixed support doctrine boolean types for subpalette checkboxes

### PR DESCRIPTION
**Pull request by Toflar**

### CODE REVIEW

File: src/Resources/contao/drivers/DC_Table.php [hunk 1]
```diff
@@ -3252,7 +3252,7 @@ public function getPalette()
 
 					if ($trigger != '')
```
Comment (qzminski) [inline src/Resources/contao/drivers/DC_Table.php]: Maybe it would be just better to have ```if ($trigger) {``` here...

**Comment by Toflar** [comment]
@qzminski I wasn't sure if it does have an effect on the `else` and I didn't want to break anything 😄  But I analyzed it more in depth now and you're right, it works exactly the same. Updated the PR, works like a charm 😄

**Comment by aschempp** [comment]
This only works for subpalettes now, right? Not for Checkbox widgets etc…

**Comment by Toflar** [comment]
> This only works for subpalettes now, right? Not for Checkbox widgets etc…

? A checkbox is a subpalette?

**Pull request merged**